src/Routes.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Router, Stack, Scene } from 'react-native-router-flux';
+import { StatusBar } from 'expo-status-bar';
+import Login from './screens/Login';
+import Register from './screens/Register';
+import Feed from './screens/Feed';
+import ForgotPassword from './screens/ForgotPassword';
+
+export default class Routes extends Component {
+  render() {
+    return (
+      <Router>
```

Comment:
Could you add a //TODO to check whether we can replace flux with react-navigation, just so that we don't forget for the next sprints